Calculator.py

```diff
@@ -17,6 +17,9 @@ def run_operation(number_1,  number_2, operation):
     elif operation == '/':
         print('{} / {} = '.format(number_1, number_2), end = '')
         return number_1 / number_2
```

Comment:
I like your idea of adding a modulo. Often overlooked useful function that can be hard to find in certain calculators.